explainer.md

```diff
@@ -186,225 +181,236 @@ https://user-images.githubusercontent.com/93594/141100217-ba1fa157-cd79-4a9d-b3b
+### Via CSS
 
-- If an element exists in Page-A only (exit animation), a fade animation takes its container from opacity 1 to 0.
-- If an element exists in Page-B only (entry animation), a fade animation takes its container from opacity 0 to 1.
-- If an element exists in both Page-A and Page-B, and both are 'container and content', an animation takes the container from Page-A styles to Page-B styles (which will include the transform used for positioning), while cross-fading the two images.
-- If an element exists in both Page-A and Page-B, and neither are 'container and child', a transform animation takes its container from Page-A size/transform to Page-B size/transform, while cross-fading the two images.
+#### Offering Page-B elements
 
-- [Open question](https://github.com/WICG/shared-element-transitions/issues/82): What if the Page-A element is 'container and child' but the Page-B element is 'single image'?
+Page-B elements are offered for the transition using the same mechanism as in Page-A, `page-transition-tag`.
 
-Because the images are sized to 100% of the container, the images will also change size throughout the transition. How these are scaled can be changed using regular CSS features like `object-fit`.
+Elements from Page-A and Page-B that have the same tag are merged into one transition elements as documented in "Mixing in elements from Page-B and associating them with transition elements from Page-A". If multiple elements have the same tag, they're associated in DOM order.
 
-In all cases, the duration and easing is some undecided default, that could even be platform independent.
+#### Targeting transition elements
 
-- [Open question](https://github.com/WICG/shared-element-transitions/issues/83): When will the default animation start? When the browser would usually first render Page-B?
-- [Open question](https://github.com/WICG/shared-element-transitions/issues/84): Default animations work well for things which are at least partially in-viewport in both Page-A and Page-B, but it gets tricky if you consider a non-sticky header that scrolled out of view by 1000s of pixels.
-- [Open question](https://github.com/WICG/shared-element-transitions/issues/85): If the developer wants a default animation of the root only, how do they define that?
+The pseudo-elements are constructed as documented in "how are transition elements represented".
 
-### CSS animation
+These pseudo-element selectors provide access to these pseudo-elements via `:root`:
 
-CSS can be used to build on default animations, or override the default.
+- `::page-transition(tag)` - Select the 'transition element' of a given `page-transition-tag`.
+- `::page-transition(tag)::image-wrapper` - Select the 'image wrapper'.
+- `::page-transition(tag)::image(outgoing | incoming)` - Select the incoming or outgoing image.
 
-```css
-::page-transition-container(header) {
-  /* … */
-}
-```
+`::page-transition` without a tag selects all transition elements.
```

Comment:
Could we use ::page-transition(*) for this instead? Mostly because I don't know if we have precedent for this. :) My reading of the parser code was that we either have selectors with no arguments or with arguments.

Reply:
`::page-transition(*)` seems fine to me!